CHANGES.txt

```diff
@@ -16,17 +16,32 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Whatever John Doe did.
 
   From Joseph Brill:
-    - MSVS: Fix a significant MSVC/MSVC tool initialization slowdown when
-      vcpkg has been installed and the PSModulePath is not initialized
-      or propagated from the user's shell environment. To resolve this
-      The default Windows Powershell 7 path is added before the default
-      Windows Powershell 5 path in the carefully constructed
-      environment in which the MSVC batch files are run.   The shell
-      environment variables and values for VCPKG_DISABLE_METRICS and
-      VCPKG_ROOT are added to the limited MSVC environment when defined.
-      At present, the VCPKG_DISABLE_METRICS and VCPKG_ROOT variables and
-      values are not propagated to the SCons environment after running
-      the MSVC batch files.
+    - MSVC: A significant delay was experienced in the windows 2022
+      and 2025 runners due to the limited environment in which the MSVC
+      batch files are invoked.  The vcpkg component installed in Visual
+      Studio invokes a powershell script when the MSVC batch files are
+      called. The limited MSVC environment 1) did not have the preferred
+      pwsh executable on the system path, 2) did not include the powershell
+      module analysis cache location (PSModuleAnalysisCachePath), and 3) did
+      not include the powershell module path (PSModulePath) which appears to
+      be responsible for the performance degradation in the windows 2022 and
+      2025 runners. The VS vcpkg component is not installed in the windows
+      2019 runner. Adding the pwsh and powershell executable paths in the
+      order discovered on the shell environment path, passing the powershell
+      module analysis cache location, and adding a limited subset of the
+      powershell module path appears to have eliminated the significant
+      delays.  In the windows runners, any one of the these three changes
+      appears to eliminite the significant delays. It is hoped that the
+      combination of all three changes will guard against significant
+      delays in other environment configurations as well.
+    - MSVC: The following shell variables and values are passed to the
+      environment in which the MSVC batch files are invoked:
```

Comment:
"environment used by SCons to initialize MSVC"..

Just want to make it clear we're not modifying `env['ENV']` here.

Reply:
Reworked all text.